server/routes/IceServerRoutes.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const app = express();
+const ServerPortRouter = express.Router();
+const iceCollectionDirectory = require('../models/IceRaidModel');
```

Comment:
Best practice is to be consistent with the naming, there's no value in calling it something different and if anything it's more confusing. `IceRaidModel` is fine.